errors/errors.go

```diff
@@ -0,0 +1,17 @@
+package errors
+
+// Cloudbeat error is an error that is desgined to have some information that
+// can help to end user to operate cloudbeat
+// for example when a cloudbeat configuration is invalid, this error will include
+// more information about what is missing/expected and maybe links to external sources as well
+type CloudbeatError struct {
```

Comment:
I liked it, since I still don't want the launcher to be aware of cloudbeat I would just rename it.
Maybe `BeaterDegradeError` / `BeaterUnhealthyError` or some other name that indicates what is about to happen once this error is returned.
Also, it should reside near the launcher IMO.

Reply:
This is good naming, will change the name.

> Also, it should reside near the launcher IMO.
This might be an issue with imports